Makefile

```diff
@@ -1,16 +1,23 @@
+IMG ?= controller:latest
+
 all: build
 
-verify: fmt vet 
+verify-%:
+	make $*
+	./hack/verify-diff.sh
```

Comment:
Just a note that in the CI, I think we need to run `verify-fmt` and `verify-vet` to make sure that once those tools have run, there's no diff left over